bee-test/src/rand/output.rs

```diff
@@ -41,26 +43,26 @@ pub fn rand_output() -> Output {
     }
 }
 
-// /// Generates a random consumed output.
-// pub fn rand_consumed_output() -> ConsumedOutput {
-//     ConsumedOutput::new(rand_transaction_id(), rand_milestone_index())
-// }
-//
-// /// Generates a random creates output.
-// // TODO finish
-// pub fn rand_created_output() -> CreatedOutput {
-//     CreatedOutput::new(
-//         rand_message_id(),
-//         match rand_number::<u64>() % 3 {
-//             0 => rand_simple_output().into(),
-//             // 1 => todo!(),
-//             _ => rand_treasury_output().into(),
-//             // _ => unreachable!(),
-//         },
-//     )
-// }
-//
-// /// Generates a random ledger treasury output.
-// pub fn rand_ledger_treasury_output() -> TreasuryOutput {
-//     TreasuryOutput::new(rand_treasury_output(), rand_milestone_id())
-// }
+/// Generates a random consumed output.
+pub fn rand_consumed_output() -> ConsumedOutput {
+    ConsumedOutput::new(rand_transaction_id(), rand_milestone_index())
+}
+
+/// Generates a random creates output.
+// TODO finish
```

Comment:
Finish now or finish later?

Reply:
Later with the bee-test refactoring PR 👍